test/cli.spec.js

```diff
@@ -84,35 +85,59 @@ describe("Configuration files", function() {
     });
 
     it("should succeed passing an existent file path to --config", async function() {
-        const { failed } = await execCliWith(
-            [VALID_SVG, "--config", "test/projects/esm/foo/custom-svglint-config.js"]
+        project_path = "test/projects/esm/foo/";
+        filename = "custom-svglint-config.js";
+        const { failed, stdout } = await execCliWith(
+            [VALID_SVG, "--config", path.join( project_path, filename ), "--debug"]
         );
+        config_file = path.join( project_path, filename ).replace( /\\/g, "\\\\" );
         expect(failed).toBeFalsy();
+        expect(stdout.replace( /\r?\n/g, "" )).toMatch( new RegExp( `Using configuration file: (.*?)${config_file}`) );
```

Comment:
I'm not a particular fan of this kind of test, the fact that the tool logs the location of a file gives no guarentee it actually does what it says. I'd ask you to revert these changes, unless you have a different motivation for the approach.

Reply:
Reverted this file.